macOS/DuckDuckGo/NewTabPage/NewTabPageCoordinator.swift

```diff
@@ -40,45 +39,42 @@ final class NewTabPageCoordinator {
         notificationCenter: NotificationCenter = .default,
         fireDailyPixel: @escaping (PixelKitEvent) -> Void = { PixelKit.fire($0, frequency: .daily) }
     ) {
+
+        let settingsMigrator = NewTabPageProtectionsReportSettingsMigrator(keyValueStore: keyValueStore)
+        let protectionsReportModel = NewTabPageProtectionsReportModel(
```

Comment:
NTP Coordinator initializes Protections Report model because it’s needed for the "NTP shown" pixel firing logic. That model is then passed to NewTabPageActionsManager as a parameter when it’s used in its user script client.